python/core/auto_generated/raster/qgsrasterdataprovider.sip.in

```diff
@@ -77,7 +77,8 @@ Base class for raster data providers.
       WriteLayerMetadata,
       ProviderHintBenefitsFromResampling,
       ProviderHintCanPerformProviderResampling,
-      ReloadData
+      ReloadData,
+      PixelDependentRendering,
```

Comment:
I don't think "PixelDepedentRendering" is a good description of this flag. Rather something like "MultiResolution" or "DpiDependentData" is a little more descriptive.